src/slic3r/GUI/Tab.cpp

```diff
@@ -1970,6 +1970,14 @@ void TabPrint::build()
         optgroup->append_single_option_line("only_one_wall_first_layer");
         optgroup->append_single_option_line("reduce_crossing_wall");
         optgroup->append_single_option_line("max_travel_detour_distance");
+        optgroup->append_single_option_line("small_area_infill_flow_compensation");
+
+        optgroup = page->new_optgroup(L("Small Area Infill Flow Compensation"), L"param_advanced");
```

Comment:
Can we change the group title to "Small Area Infill Flow Compensation(experimental)"?